### ISSUE TITLE
[bug] Client fails on block 478558: Script validation failed on tx_idx: 0, input_idx: 0: Opcode not implemented

### ISSUE THREAD

**Issue by maciejka** [keep-starknet-strange/raito#256]
In `packages/client` command:
```
scarb test --forceall --execute_scripts tests/data/full_478557.json
```
fails with:
```
Script validation failed on tx_idx: 0, input_idx: 0: Opcode not implemented
```

**Comment by supreme2580** [comment]
@maciejka can i be assigned to this

**Comment by baitcode** [comment]
I tried to reproduce that and got:

```bash
running integration tests (with scripts) ...
test tests/data/full_478557.json ... fail (gas usage est.: 0)
failures:
	tests/data/full_478557.json — Run panicked with [10465980102873265876556552398661227733856306440119630725149366969700 ('cached output was not cached'), ].
```

**Comment by maciejka** [comment]
> I tried to reproduce that and got:
> 
> ```shell
> running integration tests (with scripts) ...
> test tests/data/full_478557.json ... fail (gas usage est.: 0)
> failures:
> 	tests/data/full_478557.json — Run panicked with [10465980102873265876556552398661227733856306440119630725149366969700 ('cached output was not cached'), ].
> ```
> 
> Is this another issue? am I missing something?

This is another issue already solved in #282.

**Comment by baitcode** [comment]
Ok. I'll synchronise my branch. Could you assign me to this one, please?
